admin-dev/themes/default/scss/controllers/_login.scss

```diff
@@ -52,7 +52,6 @@
   }
 
   .flip-container {
-    height: 420px;
     margin-top: 115px;
     transform-style: preserve-3d;
```

Comment:
https://github.com/PrestaShop/PrestaShop/pull/27812/files#r816958443

Reply:
Did you test on mobile? When I tested, we needed to remove every transform style, overwise it was blurry..